src/aui-form-field/assets/aui-form-field-core.css

```diff
@@ -35,6 +36,7 @@
 
 @media (max-width: 768px) {
     .form-field {
-        padding: 20pt;
+        padding-top: 20pt;
+        padding-bottom: 20pt;
     }
 }
```

Comment:
Removes padding left and right from the form-field element. It is now on the layout column elements.